lib/abstract_format.ml

```diff
@@ -59,7 +68,14 @@ and guard_t =
   | Guard of guard_test_t list
 and guard_test_t =
   | GuardTestCall of line_t * literal_t * guard_test_t list
+  | GuardTestMapCreation of line_t * guard_assoc_t list
+  | GuardTestMapUpdate of line_t * guard_test_t * guard_assoc_t list
+  | GuardTestBinOp of line_t * string * guard_test_t * guard_test_t
   | GuardTestVar of line_t * string
+  | GuardTestLit of literal_t
+and guard_assoc_t =
```

Comment:
Could you rename `guard_assoc_t` to `guard_test_assoc_t`, please?

Reply:
Fixed!